### ISSUE TITLE
[Bug]: 'Episode' object has no attribute '_autoReload'

### ISSUE THREAD

**Issue by KnifeFed** [Kometa-Team/Kometa#895]
### Version Number

1.17.0

### What branch are you on?

master

### Describe the Bug

On one of my libraries, no matter which default overlay I'm trying to add, I end up with the error seen in the included log. It works fine on other libraries.

### Relevant Collection/Overlay/Playlist Definition

```yml
overlay_path:
  - git: PMM/overlays/audio_codec
```

### Logs

https://gist.github.com/KnifeFed/af48d7e0382cf34aef64827c1110d37e

**Comment by KnifeFed** [comment]
Turns out it was because I had `Seasons` set to `Hide` in the library advanced settings. Setting it to `Hide for single-season series` works and that serves the same purpose as `Hide` for me.

**Comment by rcdailey** [comment]
Mine is set to hide for single season and I still get this error on shows that have only 1 season.

**Comment by KnifeFed** [comment]
@rcdailey Does it work if you set it to `Show`?

**Comment by rcdailey** [comment]
Sadly no :(

**Comment by rcdailey** [comment]
My YAML:

```yml
TV Shows:
  overlay_path:
  - remove_overlays: false
  - git: PMM/overlays/resolution
    template_variables:
      use_1080p: false
      use_720p: false
      use_480p: false
  - git: PMM/overlays/audio_codec
    template_variables:
      Opus: false
      Dolby-Digital: false
      DTS: false
      DTS-ES: false
      Dolby-Digital-Plus: false
      DTS-X: true
      DTS-HD: true
      Dolby-TrueHD: true
      Dolby-Atmos: true
```

**Comment by TheSpoon98** [comment]
I was running into the same issue, however I have Seasons set as:
Library Level - Show
Single TV Series - Hide

When I change the single TV Series from Hide -> Show - Processing continues as it should with no error.

**Comment by KnifeFed** [comment]
@rcdailey Do you have any specific shows with `Season` set to `Hide`, like @TheSpoon98 mentioned?

**Comment by rcdailey** [comment]
I was not aware that you can hide it per show. So I have not explicitly set any that I know of. And it wouldn't be easy to go through each one and check.

**Comment by rcdailey** [comment]
I think it should work regardless of that setting, is my point.

**Comment by sanswit** [comment]
I also bumped into this leveraging the Audio_codec PMM overlay .yml for commentary language overlays.

I swapped my library from Hide to Show in the advance settings page in plex and it still is encountering the error above.|

I peaked over onto this thread (https://github.com/pkkid/python-plexapi/issues/945) and it looks to be some type of PlexAPI issue (now looking at Private Object flags).

Is it possible to implement their suggestion re: PlexObject flags (https://github.com/pkkid/python-plexapi/pull/946) or roll back an API version to 4.10.1 from 4.11.1?
